error-prone-contrib/src/main/java/tech/picnic/errorprone/refasterrules/ComparatorRules.java

```diff
@@ -243,18 +245,77 @@ int after(T value1, T value2) {
     }
   }
 
+  /** Prefer {@link Collections#sort(List)} over more verbose alternatives. */
+  static final class SortCollections<T extends Comparable<? super T>> {
```

Comment:
Following our naming scheme:
```suggestion
  static final class SortCollections<T extends Comparable<? super T>> {
```